02-oop-inheritance/keypoints/2_1-6/script.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Created by MaximilianoDaniel on 15/09/2015.
+ */
+var inheritPrototype = function (child, parent){
+
+    var copyOfParent = Object.create(parent.prototype);
+
+    //Is necessary to point to the childObject constructor
+    copyOfParent = child;
+    child.prototype = copyOfParent;
+};
+
+//MovieObserver can listen to some action Like play or stop
+var MovieObserver = function (listenTo) {
+    this.listenTo = listenTo || [] ;
+};
+
+MovieObserver.prototype.play = function (movie) {
+    console.log('Playing: ' + movie.title);
+};
+
+MovieObserver.prototype.stop = function (movie) {
+    console.log('Stopped: ' + movie.title);
+};
+
+var ObserverList = function () {
+  this.observerList = [];
+};
+
+ObserverList.prototype.subscribe = function (observer) {
+    this.observerList.push(observer);
+};
+
+ObserverList.prototype.unsubscribe = function (observer) {
+    var i = 0;
+
+    while( i < this.observerList.length ){
```

Comment:
not needed white spaces within () and also white space needed between while and (